src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -68,10 +77,55 @@ def _prompt_template(serialized: Dict[str, Any]) -> Iterator[Tuple[str, Any]]:
                 yield LLM_PROMPT_TEMPLATE_VARIABLES, kwargs.get("input_variables", [])
                 yield LLM_PROMPT_TEMPLATE_VERSION, "unknown"
                 break
-        except TypeError:
+        except (AttributeError, KeyError, TypeError):
+            continue
+
+
+def _invocation_parameters(extra: Dict[str, Any]) -> Iterator[Tuple[str, Any]]:
+    """Yields invocation parameters if present."""
+    for key, value in extra.get("invocation_params", {}).items():
+        yield ".".join([LLM_INVOCATION_PARAMETERS, key]), value
```

Comment:
Let's just leave it as dict (i.e. `json.dumps(extra.get("invocation_params", {}))`, because the user would need to scan the all semconv keys and look for the prefix in order to collect everything, and it's not worth the trouble.

Reply:
`json.dumps` produces a string, are you saying to have the `llm.invocation_parameters` attributes as a string?